service/src/ServiceProxy.hpp

```diff
@@ -143,6 +143,7 @@ namespace geopm
                                                 int domain,
                                                 int domain_idx,
                                                 double setting) = 0;
+            virtual std::string topo_get_cache(void) = 0;
```

Comment:
Minor note: missing code documentation. The method might be self-explanatory, but figured it was worth mentioning considering all other methods have code documentation, and in case there were any helpful notes worth mentioning.